server/config/persist_options.go

```diff
@@ -876,7 +876,16 @@ func (o *PersistOptions) getTTLUint(key string) (uint64, bool, error) {
 		return 0, false, nil
 	}
 	r, err := strconv.ParseUint(stringForm, 10, 64)
-	return r, true, err
+	if err == nil {
```

Comment:
maybe we need to provide a util about it

Reply:
transfer function name `getTTLUint` to `getTTLNumber` and add more comment